index.bs

```diff
@@ -425,7 +425,7 @@ scenarios. Additional scenarios, including sample code, are given later in [[#sc
 - Next, on their phone:
     * User sees a discrete prompt or notification, "Sign in to example.com."
     * User selects this prompt / notification.
-    * User is shown a list of their example.com identities, e.g., "Sign in as Alice / Sign in as Bob."
+    * User is shown a list of their example.com identities, e.g., "Sign in as Mohamed / Sign in as 安".
```

Comment:
We should probably get a Chinese speaker to weigh in on "安", maybe @alanwaketan? I'm also thinking we might not want both of the examples to be boy names, but I don't know whether or how 安 is gendered.

Reply:
(As a Chinese speaker) 安 is usually a family name. I have never seen a given name with the single character 安, except for the homophonic translation of the given name *Ann*.

Reply:
I searched a bit, and found a few people with the given name 安.

(Examples: [1](https://en.wikipedia.org/wiki/Ang_Lee), [2](https://www.bjyah.com/Html/Doctors/Main/Index_341.html).)

Reply:
Regarding how 安 is gendered, from the search results I saw, I think it's more used in male than in female names.

Reply:
FWIW, I think 张三 (or 小明) is usually the name for an unspecified person (like a gender-neutral version of John Doe).

Reply:
If you want a female gendered name (to balance the male gendered Mohamed), it's easy enough to search "chinese girls names". The one I tend to use in examples is 小霞 (xiaoxia) because I like the way it sounds and I have a friend who's daughter has that name. @xfq's suggestion is probably better :-)

Reply:
Thanks all. Have switched to using 张三.